packages-ui/roosterjs-react/lib/ribbon/type/RibbonPlugin.ts

```diff
@@ -20,7 +20,7 @@ export default interface RibbonPlugin extends ReactEditorPlugin {
     onButtonClick: <T extends string>(
         button: RibbonButton<T>,
         key: T,
-        strings: LocalizedStrings<T>
+        strings?: Partial<LocalizedStrings<T>>
```

Comment:
how about we change the declaration of LocalizedStrings to make everything optional, so that no need to add Partial everywhere.